pkgs/data/fonts/go-font/default.nix

```diff
@@ -1,11 +1,11 @@
 { lib, fetchgit }:
 
 let
-  version = "2017-03-30";
+  version = "2022-06-17";
```

Comment:
Shouldn't the version be 2.010 now? See https://go.googlesource.com/image/+/41969df76e82aeec85fa3821b1e24955ea993001

Reply:
I spent quite a bit of time on deciding what this should be :)
I first changed it to `2.010` as it was declared in the commit. ...But then it felt odd to declare a version where the upstream doesn't use any VCS tags. Considered maybe it should be `2.010-2022-06-17` but it was long for no good reason. Sighed and fell back to datestamp.

If you as the maintainer are OK with `2.010` directly to a commit hash, I'll happily update the PR.

Reply:
I'm in favor! I think you forgot to update the version string in the expression, though.

Reply:
git amend alias mistake. fixed.